src/main/java/spoon/support/reflect/reference/CtTypeReferenceImpl.java

```diff
@@ -465,7 +465,14 @@ private void handleParentNotFound(SpoonClassNotFoundException cnfe) {
 	public Collection<CtExecutableReference<?>> getDeclaredExecutables() {
 		CtType<T> t = getDeclaration();
 		if (t == null) {
-			return RtHelper.getAllExecutables(getActualClass(), getFactory());
+			try {
+				return RtHelper.getAllExecutables(getActualClass(), getFactory());
+			} catch (final SpoonClassNotFoundException e) {
+				if (getFactory().getEnvironment().getNoClasspath()) {
+					return Collections.emptyList();
```

Comment:
I was not sure if I should return an empty list or null. When changing this to null, there is another method to fix as it uses a for each loop without checking the type returned by this method. However, I used null since CtTypeImpl instantiates `typeMembers` with an empty list indicating that an unknown type has no methods.